charts/timescaledb-single/values.yaml

```diff
@@ -102,12 +93,13 @@ env:
 #        name: myapplication-passwords
 #        key: standby
 
-# use envFrom to expose all data of a secret/configmap
-# https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.17/#container-v1-core
+# Externally created Kubernetes secrets will be injected into the pods by referencing them here. You
+# can also add more configuration options and secrets this way (see https://kubernetes.io/docs/tasks/configure-pod-container/configure-pod-configmap/#configure-all-key-value-pairs-in-a-configmap-as-container-environment-variables)
 envFrom:
-#  - configMapRef:
-#      name: my-deployment-settings
-#      optional: true
+  - secretRef:
+      name: postgres-secrets
```

Comment:
Why not make the name of this secret an overridable value (e.g. "existingSecret").
If this value is not set, fall-back to a default of "postgres-secrets".
This would allow the use of pre-existing secrets that may be generated by an umbrella chart.

Reply:
That sounds like a reasonable approach. I'll update the PR.